pages/guide_test.py

```diff
@@ -449,3 +449,33 @@ def test_html_rendering(self):
         self.template_data, self.full_template_path)
     parser = html5lib.HTMLParser(strict=True)
     document = parser.parse(template_text)
+
+class FeatureEditAllFieldsTemplateTest(TestWithFeature):
```

Comment:
This is the same name as the preceding test class.

Reply:
Wanted to use this as a template for testing the new class, but forgot to change the most important part 😅